.github/workflows/release-please.yml

```diff
@@ -0,0 +1,14 @@
+name: release-please
+on:
+  push:
+    branches:
+      - main
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GoogleCloudPlatform/release-please-action@v2
+        with:
+          token: ${{ secrets.NODE_PKG_RELEASE_TOKEN }}
```

Comment:
Before merging the PR we'll need to add a GitHub token with `repo` permissions named `NODE_PKG_RELEASE_TOKEN` as a secret for this repo

Reply:
where do we get this token?

Reply:
done!